tox.ini

```diff
@@ -19,23 +20,9 @@ basepython =
     py35: python3.5
 
 deps =
-    django-modelcluster>=1.1b1
-    django-taggit>=0.17.5
-    django-treebeard>=3.0,<5.0
     django-sendfile==0.3.6
```

Comment:
I'd prefer to keep a distinction between "packages required in order to run the tests" and "packages required for all tests to pass without skipping". It's possible that the latter set might have some onerous installation requirements - even if that's not the case at the moment - and I wouldn't want to make life harder for developers getting up and running. (I'm not sure why elasticsearch is in the former list - maybe that was an oversight, or maybe there's something odd about the way the tests import it...)

I like the idea of having everything in setup.py though - perhaps we could have a separate 'testing_optional' list there?